ansible/roles/hadoop-ha/templates/core-site.xml

```diff
@@ -38,4 +38,34 @@
     <name>ha.zookeeper.quorum</name>
     <value>{{ zookeeper_connect }}</value>
   </property>
+{% if use_adlsg2 == True %}
+  <property>
```

Comment:
If this section is the same as the HA core site, I wonder if it would be possible to place it in a separate file and include it in both core site files.  This would ease maintenance going forward.  Maybe something like

```
  {% if use_adlsg2 == True %}
     {% include "../../hadoop/adslg2-core-site-snippet.xml" %}
  {% endif %}
```

I am not sure if this will work or if this the best way to do this.  I was just looking around at some ansible docs and saw  `include`.

Reply:
This is a great suggestion.  We will create an issue to track this change and work on it after wipe/terminate PR. Let me know if you are good with this.